### PR TITLE
Milestone 1: setup project

### PR DESCRIPTION
**Pull request by abdessalem1998**
In this pull request firstly i use it to check if the github actions and linter checker are set correctly and later for submitting for review for setting up the project:

- Set up a React app..
- Use React documentation.
- Setup linter cheker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
-# Math-Magicians
-added later
+# Getting Started with Create React App
```
Comment (SajjadAhmad14) [inline README.md]: Please add a descriptive project title and description in README. It looks more professional this way.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Dib, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
...
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
```
Comment (ZeenatLawal) [inline README.md]: - [ ] The README file for this project is not descriptive as mentioned by the previous reviewer. Kindly follow this [template](https://github.com/microverseinc/readme-template) as you've done with previous projects.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @abdessalem1998, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @abdessalem1998, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**